### ISSUE TITLE
Hanging with from block

### ISSUE THREAD

**Issue by controversial** [AtomLinter/linter-sass-lint#158]
- Atom version: 1.10.2
- linter-sass-lint version: 1.7.1
- Global sass-lint version (if used): `which sass-lint` does report something but I'm not sure how to check the version.
- Platform:
  - [x] OSX
  - [ ] Windows
  - [ ] Linux
- Expected behaviour: `liner-sass-lint` lints my `.sass` file
- Actual behaviour / issue description:
  When typing `from` in a `@keyframes` block using the tab-indented syntax, Atom hangs.
  ### Steps to reproduce:
- Create a new `.sass` file
- Type:

``` sass
@keyframes
  from
```

Upon pressing "enter" after "from," Atom will hang. Sometimes I have to delete the whitespace in the new line for Atom to hang, but it always hangs somehow.

I have "as-you-type linting" turned on.

**Comment by Arcanemagus** [comment]
If you save the file in the state that hangs Atom, and then run `sass-lint` on it from the CLI, does it take an inordinately long amount of time as well?

**Comment by DanPurdy** [comment]
I'm going to go ahead and assume it's the AST we use getting itself into an infinite loop. Let me check it when I'm home and I'll let you know..

**Comment by controversial** [comment]
Ok, thanks @DanPurdy. Keep us posted.

**Comment by DanPurdy** [comment]
Long time ago but I believe this is fixed in the latest versions of gonzales-pe and consequently sass-lint

```
@keyframes
  from
    margin-left: 100%
    width: 300

  to
    margin-left: 0%
    width: 100%
```

not crashing and definitely not falling over.
